tag_browser_plugin/tag_browser_plugin.go

```
@@ -0,0 +1,116 @@
+	}
+
+	message := service.NewMessage(nil)
+	message.SetBytes(protoBytes)
+	resultBatch = append(resultBatch, message)
+
+	return []service.MessageBatch{resultBatch}, nil
+}
+
+func (t TagBrowserProcessor) Close(_ context.Context) error {
+	// Wipe cache
+	t.unsMapCache = make(map[string]bool)
```

golangci-lint (1.64.8)

SA4005: ineffective assignment to field TagBrowserProcessor.unsMapCache

(staticcheck)